### PR TITLE
Add dockerfile and docker-compose to react project

### PR DESCRIPTION
**Pull request by shadman0711**
Fixes #

- [x] Does not depend on other PRs
- [ ] Depends on other PR being merged first: #

## What kind of change does this PR introduce (check at least one)

- [ ] Bugfix
- [ ] Improvement
- [ ] New feature
- [ ] Refactor
- [x] Infrastructure
- [ ] Other (please specify)

## How would you describe the changes in a changelog (1-2 lines)?
Dockerfile added in the react js project

## What was done
* Dockerfile added
* dockerignore file added

## How to test

- [ ] Needs regression testing (add QA needed label)

## Deploy risk & follow up

Please assess risk & follow up on deployment of this PR. Elaborate your choice if selecting anything other than 'No risk'.

- [x] No risk
- [ ] Low
- [ ] High

## Reminder list

- Determine whether REST API Documentation should be updated
- Make an assessment on whether unit tests should be updated and/or added
- Ensure follow up tasks are linked to this PR

QA - if regression testing was requested:

- [ ] QA has verified

### CODE REVIEW

File: react-kanban/Dockerfile.prod [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# stage-1: build the app
+FROM node:alpine as builder
+WORKDIR /app
+COPY package.json ./
+COPY package-lock.json ./
```
Comment (musfique-m) [inline react-kanban/Dockerfile.prod:5]: Just a reminder, you can reduce the number of layer by using wildcard (*) here.
`COPY package*.json ./`

**Review by musfique-m — APPROVED** [review]
Looks good.

File: react-kanban/Dockerfile.prod (hunk 1 above)
Comment (shadman0711) [reply react-kanban/Dockerfile.prod:5]: Updated Dockerfile in commit [45a065d](https://github.com/CraftsmenLtd/coe-batch-q322/pull/280/commits/45a065d06e5a969b305d3e5432f479487fbf8b8d)

Comment (musfique-m) [reply react-kanban/Dockerfile.prod:5]: Please also update this file.

**Review by musfique-m — CHANGES_REQUESTED** [review]

**Comment by shadman0711** [comment]
Added wildcard in Dockerfile.prod in commit [149bbd1](https://github.com/CraftsmenLtd/coe-batch-q322/pull/280/commits/149bbd100d1f2dd1ff3943e0f0a5e82d10497714)

**Review by musfique-m — APPROVED** [review]
Looks good.

**Pull request merged**